.github/workflows/ros1.yml

```diff
@@ -41,6 +41,8 @@ jobs:
       run: |
         # TODO(ros-tooling/setup-ros-docker#7): calling chown is necessary for now
         sudo chown -R rosbuild:rosbuild "$HOME" .
+    - name: test-git-secret-actions
+      uses: aws-robotics/aws-robomaker-github-actions/git-secrets-scan-action@2.3.3
```

Comment:
Same comments as those in https://github.com/aws-robotics/aws-robomaker-sample-application-helloworld/pull/94.